### DISCUSSION TITLE
Architecture issue

### DISCUSSION THREAD

**Discussion by ma1den** [farag2/Sophia-Script-for-Windows#170] [Q&A]
Hello everyone,
This script is great. I am using it some time and no problems at all.
Yesterday I try to run it at pc which report error: Import-Module :
```The current processor architecture is: X86.  The module
'\Windows-10-Sophia-Script-master\Sophia\PowerShell 5.1\Sophia.psd1' requires the following architecture: Amd64.
```
What I can do to execute script on this pc?!

**Comment by farag2** [comment]
Noway. Is the CPU (may be OS?) architecture really x86?

run this code from admin

```powershell
[System.Environment]::Is64BitOperatingSystem
```

What you get, `true` or `false`?

May be you launched `powershell x86` instead of `powershell`?

**Reply by ma1den** [reply]
OS is x86. 
Running this command I get false.
I will try to runn poweshell x86

**Reply by farag2** [reply]
No. If the OS architecture is x86 the script won't work at all. The scrip supports Windows 10 x64 only.

**Reply by ma1den** [reply]
Yes. I get same error.
```
Import-Module : The current processor architecture is: X86.  The module
'\Windows-10-Sophia-Script-master\Sophia\PowerShell 5.1\Sophia.psd1' requires the following architecture: Amd64.
```
Thank you for quick reply.

**Reply by farag2** [reply]
Of course... It's time for the PC upgrade!

**Reply by ma1den** [reply]
Yes.
